x-pack/docs/en/security/authentication/configuring-native-realm.asciidoc

```diff
@@ -1,6 +1,13 @@
-The native realm is available by default when no other realms are 
-configured. If other realm settings have been configured in `elasticsearch.yml`, 
-you must add the native realm to the realm chain.
+The native realm is available and enabled by default. If you want to use a different method to manage and authenticate users you need to disable the native realm. 
+
+The following snippet displays an example of how to disable the `native` realm.
```

Comment:
This does not read right to me. We don't need disable native realm to use "a different method to manage and authenticate users". They can co-exist. I think we probably just need to say

```suggestion
The native realm is available and enabled by default. You can disable it explicitly with the following snippet. 
```